rmf_fleet_adapter_python/src/adapter.cpp

```diff
@@ -116,8 +133,43 @@ PYBIND11_MODULE(rmf_adapter, m) {
              py::arg("profile"),
              py::arg("start"),
              py::arg("handle_cb"))
+        .def("set_task_planner_params", 
+             [&](agv::FleetUpdateHandle& self,
+                battery::BatterySystem& b_sys,
+                battery::SimpleMotionPowerSink& m_sink,
+                battery::SimpleDevicePowerSink& a_sink,
+                battery::SimpleDevicePowerSink& t_sink)
+             {
+                return self.set_task_planner_params(
+                    std::make_shared<battery::BatterySystem>(b_sys),
+                    std::make_shared<battery::SimpleMotionPowerSink>(m_sink), 
+                    std::make_shared<battery::SimpleDevicePowerSink>(a_sink),
+                    std::make_shared<battery::SimpleDevicePowerSink>(t_sink));
+             },
+             py::arg("battery_system"),
+             py::arg("motion_sink"),
+             py::arg("ambient_sink"),
+             py::arg("tool_sink"))
+        .def("account_for_battery_drain",
+             &agv::FleetUpdateHandle::account_for_battery_drain,
+             py::arg("value") = true)
+        .def("set_recharge_threshold", 
+             &agv::FleetUpdateHandle::set_recharge_threshold,
+             py::arg("threshold"))
+        // NOTE: deprecated 
         .def("accept_delivery_requests",
-             &agv::FleetUpdateHandle::accept_delivery_requests);
+             &agv::FleetUpdateHandle::accept_delivery_requests,
+             "NOTE: deprecated, use accept_task_reqeusts() instead")
+        .def("accept_task_requests",
+             &agv::FleetUpdateHandle::accept_task_requests,
+             py::arg("check"),
+             "Provide a callback function which will accept rmf tasks requests")
+        .def_property("default_maximum_delay",
+             py::overload_cast<>(
+                 &agv::FleetUpdateHandle::default_maximum_delay, py::const_),
+             [&](agv::FleetUpdateHandle& self){
+                 return self.default_maximum_delay();
+             });
```

Comment:
We should also add `fleet_state_publish_period()`
https://github.com/osrf/rmf_core/blob/8d2bea5f5bd222e0ead74540258988e4c30fc074/rmf_fleet_adapter/include/rmf_fleet_adapter/agv/FleetUpdateHandle.hpp#L162-L166